### DISCUSSION TITLE
Save plots with high resolution, nonvisible axes and background transparency?

### DISCUSSION THREAD

**Discussion by HarrisonLeece** [alandefreitas/matplotplusplus#212] [Q&A] [answered]
I looked through the Docs but couldn't find sections on making 'aesthetic' (as opposed to functional) plot renders.

I would like to render the plot at 1920x1080p, with no axes and alpha transparency in the background.  What options do I have for making this happen? 

Thank you

E: Size can be handled 
```
 auto fig = figure(true);
 fig->size(1920,1080);
```

**Comment by alandefreitas** [comment, marked as answer]
Yes. You need to take the particular figure, axes, or axes_object handles and change the properties you want. There's not much to learn. 

The documentation has a section on plot [appearance](https://alandefreitas.github.io/matplotplusplus/appearance/labels/title/), and another on [exporting](https://alandefreitas.github.io/matplotplusplus/exporting/saving-manually/) the images but it's impossible to cover everything that is possible. If the option is not in the documentation, the only option is to look at the appropriate header file (figure, axes, axes_object), depending on what object you want to edit.

**Reply by HarrisonLeece** [reply]
Re: exporting, many images on the github page have a transparent background.  Could you save me some time by telling me how that was achieved?  If I can do that, I might choose to programmatically crop the axes from the images I instead of trying to figure out a process that works in the library.  My C++ competency is too low to comprehend header files at any appreciable speed.

Thanks Alan

**Reply by alandefreitas** [reply]
There's nothing special in these images. I build the lib with an option to save all figures when the program is over and I run all examples with this [script](https://github.com/alandefreitas/matplotplusplus/blob/master/test/generate_examples/main.cpp).

**Reply by HarrisonLeece** [reply]
It looks like .SVG saved plots always have transparent backgrounds.

In my case: I was able to convert .svg to a more suitable file format using inkscape CLI.  Because I had 600 images in an animation I wanted to convert, I used a for loop in a bash script.  I include it below:

```
for i in $(seq 0 1 599 $END); 
	do 
		echo $i;
		inkscape --export-filename="./tempPNG/waveform${i}.png" --export-background-opacity=0 --export-type=png "./plot_saves/flaggerWaveform${i}.svg"
done
```
